prisondersdilemma.py

```diff
@@ -11,182 +11,260 @@
 from numpy.random import shuffle
 import numpy as np
 import random
+from win32api import GetSystemMetrics
```

Comment:
This is to obtain window dimensions if I'm not mistaken? I'm unfamiliar but is it supported across all operating systems?